client/src/components/Minimap.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <div class="container">
         <div class="area-title">
-            <div class="map-name">{{ areaTitle }}</div>
+            <div class="map-name" @click="handleAreaClick">{{ areaTitle }}</div>
```

Comment:
Can you also make it so `.map-name:hover` has `cursor: pointer` within the styles for this component, to indicate that it can be clicked on?